### PR TITLE
SDL_UCS4ToUTF8 may exist from SDL_keyboard.c

### PR DESCRIPTION
**Pull request by pmp-p**
test CI for removing function

### CODE REVIEW

**Review by ankith26 — COMMENTED** [review]
Another alternative is to just use the `pg_` prefix for this, and not rely on SDL at all on WASM (since this is an internal SDL util function, might not be wise to do)

**Comment by pmp-p** [comment]
i think using the internal may gain some space ( .wasm are not compressed unlike .data) it's worth the risk and easy to fix later since function code will stay just above.

**Comment by Starbuck5** [comment]
What about using `#ifndef` to define the function on our end only if it hasn't already been defined?

**Comment by ankith26** [comment]
That would not work because `SDL_UCS4ToUTF8` is a function name not a macro

**Review by ankith26 — APPROVED** [review]
LGTM! Thanks 😎 

This has no affect on our non-static builds, but if this makes things better for WASM lets merge this in

**Review by MyreMylar — APPROVED** [review]
LGTM 👍

File: src_c/key.c [hunk 1]
```diff
@@ -439,6 +439,11 @@ static const char *SDL1_scancode_names[SDL_NUM_SCANCODES] = {
...
 };
 
+#if defined(BUILD_STATIC)
+// from SDL_keyboard.c
+extern char *
+SDL_UCS4ToUTF8(Uint32 ch, char *dst);
+#else
 /* Taken from SDL_iconv() */
 char *
 SDL_UCS4ToUTF8(Uint32 ch, char *dst)
```
Comment (illume) [inline src_c/key.c:449]: ps. We should never define our own "SDL_" functions. This has bitten us in the past. Instead we should create pg_XXX functions which inside may call the SDL_XXX function, or include the function body. This way if someone links something with the SDL_XXX function in the future we have no problems.

**Review by illume — COMMENTED** [review]
👍

**Pull request merged**